Doc/library/os.rst

```diff
@@ -2366,8 +2366,7 @@ features:
 
    .. attribute:: st_ino
 
-      Inode number. On Windows, this is an arbitrary value that uniquely
-      identifies the file on its device.
+      Inode number.
```

Comment:
Could we switch this to "Inode number on UNIX, file index on Windows" and add the MSDN link here instead of in the what's new section?